TypeCobol.Test/Parser/Programs/Cobol85/DebugNormalLines.rdz.cbl

```diff
@@ -57,6 +63,7 @@
            MOVE xxxx TO yyyy
       d    DISPLAY 'name = '
                 yyyy
+      d    DISPLAY 'name = ' yyyy
            MOVE '1234' TO yyyy
```

Comment:
@delevoye 
Can you add the following line after line 67 (just before GOBACK) ?
```cob
d    DISPLAY 
     'name = ' yyyy
```

Reply:
The line has been added and the corresponding test updated accordingly.

Reply:
@delevoye 
After checking we should have an error on line 69 because `'name = ' yyyy` doesn't belong to any valid COBOL statement.

Reply:
Line 69 is `END PROGRAM Pgm2.`, do you mean line 59 ?
Without debugging mode, Pgm2 is equivalent to:
```Cobol
           MOVE xxxx TO yyyy
                yyyy
           MOVE '1234' TO yyyy
           GOBACK.
       END PROGRAM Pgm2.
```
which is valid from a Cobol point of view.

Reply:
@delevoye @smedilol 
The following code has disappeared from the test. That's why we can't see it.
```cob
d    DISPLAY 
     'name = ' yyyy
```

Reply:
As you've seen, tests have been splitted and the diagnostics updated.